### PR TITLE
don't default stage to last saved.

### PR DESCRIPTION
**Pull request by CeciliaHwang**
add "- select -" to stage list. if stage is not selected, show message.

### CODE REVIEW

**Comment by eduardb** [comment]
Hey @CeciliaHwang, if you could rebase and fix the conflict, I could merge for you.

**Comment by CeciliaHwang** [comment]
Hey @eduardb I rebased and resolved the conflict. Let me know if you notice any other issues. Thank you.

File: app/src/main/java/net/aiscope/gdd_app/ui/metadata/MetadataActivity.kt [hunk 1]
```diff
@@ -104,12 +102,17 @@ class MetadataActivity : AppCompatActivity() , MetadataView, CaptureFlow {
...
-            presenter.save(
-                metadata_section_smear_type_radio_group.checkedRadioButtonId,
-                metadata_species_spinner.selectedItem.toString(),
-                metadata_stage_spinner.selectedItem.toString()
-            )
+        var stage = metadata_stage_spinner.selectedItem.toString()
+        if (stage == this.baseContext.getString(R.string.malaria_stage_prompt)) {
+            metadata_stage_error.text = this.baseContext.getString(R.string.malaria_stage_required)
+        } else {
+            coroutineScope.launch {
```
Comment (eduardb) [inline app/src/main/java/net/aiscope/gdd_app/ui/metadata/MetadataActivity.kt:109]: I think that this kind of logic should be in the presenter itself, no?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -52,12 +52,15 @@
...
 
+    <string name="malaria_stage_required">Select a stage</string>
+    <string name="malaria_stage_prompt">- select -</string>
     <string name="malaria_stage_ring">Ring</string>
     <string name="malaria_stage_trophozoite">Trophozoite</string>
     <string name="malaria_stage_schizont">Schizont</string>
     <string name="malaria_stage_gametocyte">Gametocyte</string>
     <string name="profile">profile</string>
     <string-array name="metadata_malaria_stages">
+        <item>- select -</item>
```
Comment (eduardb) [inline app/src/main/res/values/strings.xml:63]: Duplication could be removed here by using the strings defined above.

**Pull request merged**